packages/wp-dashboard/src/components/editorSettings/customFonts/index.js

```diff
@@ -33,14 +33,14 @@ import {
   Icons,
   Text,
   THEME_CONSTANTS,
-  Tooltip,
   themeHelpers,
   useSnackbar,
   useLiveRegion,
 } from '@googleforcreators/design-system';
 
 import styled from 'styled-components';
 import { trackEvent, trackError } from '@googleforcreators/tracking';
+import { Tooltip } from '@googleforcreators/dashboard';
```

Comment:
Will this cause a circular dep? 🤔

Reply:
it shouldn't. The editor is getting imported to wp-editor.